cmd/artictl/utils.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
This file contains functions related to context, so the better name will be `context.go` or `context_utils.go` (to find it easy in source code).